.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "2.2.0"
```

Comment:
That's a lot of places to update versions. This done automatically or manually?

Reply:
That is done automatically by the release-please action :).

Reply:
What I hoped for, nice :)